Docs/Configuration.tex

```diff
@@ -6665,8 +6654,8 @@ \subsection{Quirks Properties}\label{uefiquirkprops}
   This is a very rough workaround to circumvent the \texttt{Still\ waiting\ for\ root\ device} message
   on some APTIO IV firmware (ASUS Z87-Pro) particularly when using FileVault 2.
   It appears that for some reason, they execute code in parallel to \texttt{EXIT\_BOOT\_SERVICES},
-  which results in the SATA controller being inaccessible from macOS. A better approach should be
-  found in some future. Expect 3 to 5 seconds to be adequate when this quirk is needed.
+  which results in the SATA controller being inaccessible from macOS. A better approach would be
```

Comment:
Actually no such intent. It is rather, if anybody knows a better way, report and help us implement it.

Reply:
How about ...
`A better approach is required and Acidanthera is open to suggestions.`

Reply:
Fine to me.